posthog/tasks/exporter.py

```diff
@@ -120,6 +123,37 @@ def _export_to_png(exported_asset: ExportedAsset) -> None:
             driver.close()
 
 
+def _get_filter_by_day(filter: Filter) -> List[Filter]:
+    # split this filter in to multiple single day filters
+    # otherwise the data is too large for ClickHouse to handle
+    return []
+
+
+def stage_results_to_object_storage(day_filter: Filter) -> str:
+    # load results
+    # write them to object storage
+    # return the path of the new file
+    return "i am a path"
+
+
+def concat_results_in_object_storage(day_files: List[str]) -> None:
+    # each of the files can be loaded in order and concatenated
+    # to make one big CSV
+    # and then the asset updated?
+    pass
+
+
+def _export_to_csv(exported_asset: ExportedAsset) -> None:
+    export_context = json.loads(exported_asset.export_context)
+    if export_context.get("type", None) == "list_events":
+        filter = Filter(data=export_context.get("filter"))
+        day_files = []
+        for day_filter in _get_filter_by_day(filter):
```

Comment:
Maybe one thing we can add here when you get round to the details, is a check as to whether the CSV actually _needs_ to be split up or if it can't just be done in memory. Might be that the 90th percentile of exports always have one file anyways.